app/workers/warehouse_dump_worker.rb

```diff
@@ -0,0 +1,41 @@
+require 'csv'
+
+class WarehouseDumpWorker
+  include Sidekiq::Worker
+  include Sidetiq::Schedulable
+
+  recurrence(backfill: true) { daily.hour_of_day(1) }
+
+  def perform(last_occurence, current_occurence)
+    from = Time.at(last_occurence)
```

Comment:
Do not use `Time.at` without zone. Use one of `Time.zone.at`, `Time.at.current`, `Time.at.in_time_zone`, `Time.at.utc`, `Time.at.getlocal`, `Time.at.iso8601`, `Time.at.jisx0301`, `Time.at.rfc3339`, `Time.at.to_i`, `Time.at.to_f` instead.